RecoBTag/CTagging/interface/CharmTagger.h

```diff
@@ -0,0 +1,63 @@
+#ifndef RecoBTag_CTagging_CharmTagger_h
+#define RecoBTag_CTagging_CharmTagger_h
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "CommonTools/Utils/interface/TMVAEvaluator.h"
+#include "RecoBTau/JetTagComputer/interface/JetTagComputer.h"
+#include <mutex>
+#include "FWCore/Utilities/interface/ESInputTag.h"
+#include "RecoBTag/SecondaryVertex/interface/CombinedSVSoftLeptonComputer.h"
+#include "DataFormats/BTauReco/interface/TaggingVariable.h"
+#include "RecoBTau/JetTagComputer/interface/JetTagComputerRecord.h"
+
+//DEBUGGING
+#include "TFile.h"
```

Comment:
please remove the TFile and TNtuple
If really needed for debugging, hide them under ifdefs to be picked up by recompilation